private/coursier_utilities.bzl

```diff
@@ -28,18 +28,49 @@ SUPPORTED_PACKAGING_TYPES = [
 ]
 
 def strip_packaging_and_classifier(coord):
-    # We add "pom" into _COURSIER_PACKAGING_TYPES here because "pom" is not a
```

Comment:
I originally implemented this as
```
def strip_packaging_and_classifier(coord):
    # Strip packaging and classifier from the following maven coordinate formats
    # groupId:artifactId:version
    # groupId:artifactId:packaging:version
    # groupId:artifactId:packaging:classifier:version
    coordinates = coord.split(":")
    return ":".join([coordinates[0], coordinates[1], coordinates[-1]])
```
but stripping all classifiers and types causes problems for resolving some artifacts so I went back to the more selective removal of classifiers and package types

Reply:
SGTM